src/components/calculator.tsx

```diff
@@ -192,6 +236,35 @@ export default function Calculator() {
         >
           ln
         </button>
+        {calculatorMode === CalculatorMode[1] && (
+          <>
+            <button
+              onClick={handleSin}
+              className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
+            >
+              sin
+            </button>
+            <button
+              onClick={handleCos}
+              className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
+            >
+              cos
+            </button>
+            <button
+              onClick={handleTan}
+              className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
+            >
+              tan
+            </button>
+            <button
+              onClick={handleLogToE}
+              className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
+            >
+              arcsin
+            </button>
```

Comment:
The buttons here has the same styles as your `CalcButton` that you created, right?
What if you used that here?

Reply:
I have refactored both the buttons and the trig functions

Reply:
BEAST!!!!!!!